cooldowns_evoker.lua

```diff
@@ -0,0 +1,390 @@
+-- ================ EVOKER ================
+
+local SPEC_EVOKER_DEVASTATION = 1467
+local SPEC_EVOKER_PRESERVATION = 1468
+
+-- Evoker/Baseline
+-- Fire Breath
+LCT_SpellData[357208] = {
+  class = "EVOKER",
+  cooldown = 30,
+  -- TODO: Fire Breath's CD is reduced when consuming essence bursts when
+  --       talented into 'Feed the Flames'
+}
+
+-- Blessing of the Bronze
+LCT_SpellData[364342] = {
+  class = "EVOKER",
+  cooldown = 15
+}
+
+-- Hover
+LCT_SpellData[358267] = {
+  class = "EVOKER",
+  cooldown = 35,
+  opt_charges = 2 -- Hover gets two charges with the 'Aerial Mastery' talent
+}
+
+-- Living Flame
+LCT_SpellData[361469] = {
+  class = "EVOKER",
+  heal = true,
+}
+
+-- Deep Breath
+LCT_SpellData[357210] = {
+  class = "EVOKER",
+  cooldown = 120,
+  opt_lower_cooldown = 60 -- Reduced by the 'Onyx Legacy' talent
+}
+
+-- Fury of the Aspects
+LCT_SpellData[390386] = {
+  class = "EVOKER",
+  cooldown = 300
+}
+
+-- Disintegrate
+LCT_SpellData[356995] = {
+  class = "EVOKER"
+}
+
+-- Azure Strike
+LCT_SpellData[362969] = {
+  class = "EVOKER"
```

Comment:
Why does this not have a cooldown?

Reply:
I blindly put all the spells. Should I take them out?

Reply:
The only time I tend to do that is when that spell has special properties (for example, fireball reducing combustion cooldown, or spells reducing hammer of justice cooldown). And I add a `hidden = true` property so they’re not displayed in the options’ cooldown list.